src/background.ts

```diff
@@ -167,34 +178,43 @@ const onStartUp = async () => {
 
   if (browser.contextMenus) {
     ContextMenuEvents.menuInit(store.getState());
-    if (!browser.contextMenus.onClicked.hasListener(ContextMenuEvents.onContextMenuClicked)) {
-      browser.contextMenus.onClicked.addListener(ContextMenuEvents.onContextMenuClicked);
+    if (
+      !browser.contextMenus.onClicked.hasListener(
+        ContextMenuEvents.onContextMenuClicked,
+      )
+    ) {
+      browser.contextMenus.onClicked.addListener(
+        ContextMenuEvents.onContextMenuClicked,
+      );
```

Comment:
Not sure if we can have prettier config change so it doesn't quite look like this?  That or I just need to try and get used to it somehow.

Reply:
Eh I trust its defaults.